src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -0,0 +1,638 @@
+    }
+}
+
+inline std::istream& operator>>(std::istream& is, Level& level) {
+    std::string str;
+    is >> str;
+    if (str == "NO") {
+        level = Level::NO;
+    } else if (str == "LOG_ERROR") {
+        level = Level::ERR;
+    } else if (str == "LOG_WARNING") {
+        level = Level::WARNING;
+    } else if (str == "LOG_INFO") {
+        level = Level::INFO;
+    } else if (str == "LOG_DEBUG") {
+        level = Level::DEBUG;
+    } else if (str == "LOG_TRACE") {
+        level = Level::TRACE;
+    } else {
+        throw ov::Exception{"Unsupported log level: " + str};
+    }
+    return is;
+}
+/** @endcond */
+
+/**
+ * @brief the key for setting desirable log level.
+ */
+static constexpr Key<Level> level{"LOG_LEVEL"};
+}  // namespace log
+
+/**
+ * @brief This key defines the directory which will be used to store any data cached by plugins.
+ *
+ * The underlying cache structure is not defined and might differ between OpenVINO releases
+ * Cached data might be platform / device specific and might be invalid after OpenVINO version change
+ * If this key is not specified or value is empty string, then caching is disabled.
+ * The key might enable caching for the plugin using the following code:
+ *
+ * @code
+ * ie.set_property("GPU", ov::cache_dir("cache/")); // enables cache for GPU plugin
+ * @endcode
+ *
+ * The following code enables caching of compiled network blobs for devices where import/export is supported
+ *
+ * @code
+ * ie.set_property(ov::cache_dir("cache/")); // enables models cache
+ * @endcode
+ */
+static constexpr Key<std::string> cache_dir{"CACHE_DIR"};
+
+namespace device {
+
+/**
+ * @brief the key for setting of required device to execute on
+ * values: device id starts from "0" - first device, "1" - second device, etc
+ */
+static constexpr Key<std::string> id{"DEVICE_ID"};
+
+/**
+ * @brief Type for device Priorities config option, with comma-separated devices listed in the desired priority
+ */
+struct Priorities : public Key<std::string> {
+private:
+    template <typename H, typename... T>
+    static inline std::string concat(const H& head, T&&... tail) {
+        return head + std::string{','} + concat(std::forward<T>(tail)...);
+    }
+
+    template <typename H>
+    static inline std::string concat(const H& head) {
+        return head;
+    }
+
+public:
+    using Key<std::string>::Key;
+
+    /**
+     * @brief Constructs device priorities
+     * @tparam Args property constructor arguments types
+     * @param args property constructor arguments
+     * @return Pair of string key representation and type erased paramter.
+     */
+    template <typename... Args>
+    inline std::pair<std::string, Any> operator()(Args&&... args) const {
+        return {str(), Any{concat(std::forward<Args>(args)...)}};
+    }
+};
+
+/**
+ * @brief Device Priorities config option, with comma-separated devices listed in the desired priority
+ */
+static constexpr Priorities priorities{"MULTI_DEVICE_PRIORITIES"};
```

Comment:
@myshevts will you add new keys `AUTO_BATCH_DEVICE_CONFIG` of this property can be reused?

@apankratovantonp what is about and `AUTO_BATCH_TIMEOUT`?

Reply:
I am going to replace devices list options in all aggregated plugins with a single property in https://github.com/openvinotoolkit/openvino/pull/9379.  
AUTO_BATCH_TIMEOUT will be added

Reply:
Confirm with @myshevts

Reply:
maxim is agree with `ov::priorities("CPU(5)")` for AUTO BATCH